app/upgrades/v8_2/constants.go

```diff
@@ -0,0 +1,10 @@
+package v82
+
+const (
+	// UpgradeName is the shared upgrade plan name for mainnet and testnet
+	UpgradeName = "v8.2.0"
+	// MainnetUpgradeHeight defines the Evmos mainnet block height on which the upgrade will take place
+	MainnetUpgradeHeight = 4_716_200
+	// UpgradeInfo defines the binaries that will be used for the upgrade
+	UpgradeInfo = `'{"binaries":{"darwin/arm64":"https://github.com/evmos/evmos/releases/download/v8.2.0/evmos_8.2.0_Darwin_arm64.tar.gz","darwin/x86_64":"https://github.com/evmos/evmos/releases/download/v8.2.0/evmos_8.2.0_Darwin_x86_64.tar.gz","linux/arm64":"https://github.com/evmos/evmos/releases/download/v8.2.0/evmos_8.2.0_Linux_arm64.tar.gz","linux/x86_64":"https://github.com/evmos/evmos/releases/download/v8.2.0/evmos_8.2.0_Linux_x86_64.tar.gz","windows/x86_64":"https://github.com/evmos/evmos/releases/download/v8.2.0/evmos_8.2.0_Windows_x86_64.zip"}}'`
```

Comment:
@fedekunze it was pointed out to us by one of the validators that their upgrade with Cosmovisor failed due to the proposal containing wrong GOARCH values. You can find the supported ones here: https://go.dev/doc/install/source#environment

To conform to the Go standard for OS architectures, we should change 

```
"linux/x86_64": "https://github.com/evmos/evmos/releases/download/v8.1.0/evmos_8.1.0_Linux_x86_64.tar.gz"
``` 

to 

```
"linux/amd64": "https://github.com/evmos/evmos/releases/download/v8.1.0/evmos_8.1.0_Linux_x86_64.tar.gz"
```